code/load_db.py

```diff
@@ -0,0 +1,69 @@
+import pyodbc
+import teardown as db
+import extract_transform as dp
+
+
+def main():
+    #For rerun purposes (incase of new daily feed)
+    db.dbcleanup()
+
+    #Extract Json and transform
+    dataframe = dp.json_to_df()
+
+    # DB_Connection
+    conn = pyodbc.connect(
+        'DRIVER={ODBC Driver 17 for SQL Server};TrustServerCertificate=No;DATABASE=Movies_DB;WSID=LAPTOP-BLDSMT2E;APP={Microsoft® Windows® Operating System};Trusted_Connection=Yes;SERVER=(localdb)\MSSQLLocalDB;Description=movies')
```

Comment:
Why do you hardcode the DB connection string?

Reply:
moved configuration to dev.ini